tests/ansible.cfg

```diff
@@ -1,4 +1,3 @@
 [defaults]
-stdout_callback = yaml
```

Comment:
I think the YAML output is much easier to read when e.g. debugging failed CI jobs... Personally i would leave this in. Not a blocker but unless there's a reason to avoid it, i think we may want to put this back in with upcoming test suite tweaks...